dev_docs/tutorials/saved_objects.mdx

```diff
@@ -19,6 +19,7 @@ import { SavedObjectsType } from 'src/core/server';
 export const dashboardVisualization: SavedObjectsType = {
   name: 'dashboard_visualization', [1]
   hidden: true,
+  switchToModelVersionAt: '8.10.0',
```

Comment:
It's good to call out the default switch is at 8.10, but I don't think model version authors should need to declare this everywhere (since it's already the default).

Maybe we can do something like:

```suggestion
  switchToModelVersionAt: '8.10.0', // this is the default, feel free to omit it unless you intend to switch to using model versions before 8.10.0
```

Or remove it entirely, what do you think?

Reply:
I guess we don't need to add the comment, since we're way past 8.10 already.